docs/reference/http-api.md

```
@@ -92,6 +92,60 @@ curl -X POST http://localhost:3000/publish \
+}
+```
+
+Plugins that haven't yet declared `publishes` / `subscribes` show up with empty arrays — they're still loaded; they just don't contribute to the topic index.
+
+---
+
+## WS /api/bus/subscribe
+
+External processes can join the bus over a WebSocket to observe topic traffic. See [external-bus-subscribers](../guides/external-bus-subscribers.md) for the full guide.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)